libs/blocks/gnav/gnav.js

```diff
@@ -421,14 +421,29 @@ class Gnav {
   };
 
   decorateSignIn = (blockEl, profileEl) => {
+    const profileDropDown = blockEl.querySelector(':scope > div:nth-child(2)');
     const signIn = blockEl.querySelector('a');
+
     signIn.classList.add('gnav-signin');
     signIn.setAttribute('daa-ll', 'Sign In');
-    profileEl.append(signIn);
-    profileEl.addEventListener('click', (e) => {
-      e.preventDefault();
-      window.adobeIMS.signIn();
-    });
+
+    if (profileDropDown) {
+      const id = `navmenu-${blockEl.className}`;
+
+      profileDropDown.id = id;
+      profileEl.classList.add('gnav-navitem', 'has-Menu');
```

Comment:
Lowercase `has-menu`

Reply:
@meganthecoder `has-Menu` is referenced in multiple places in the css.

https://github.com/adobecom/milo/blob/1fc340e1f10e929e539ca1df8fed6b460f20c20d/libs/blocks/gnav/gnav.css#L400

Reply:
Seems like a simple find and replace. Unless you're saying that the class comes from outside the Milo code. In that case, I understand that you can't change it

Reply:
This is a hold over from our pre style lint days. I'm fine with either fixing or leaving.

Reply:
I updated this.